projects/components/src/filter-button/filter-button.service.ts

```diff
@@ -37,17 +45,29 @@ export class FilterButtonService {
     this.setUrlFilters([...filters]);
   }
 
+  public removeUrlFilter(attributes: FilterAttribute[], filter: Filter): void {
+    const filters = this.getUrlFilters(attributes);
+
+    const foundIndex = filters.findIndex(f => filter.field === f.field);
```

Comment:
Why findIndex + splice instead of a plain filter? As far as I can tell, this whole method just collapses down to:
`this.setUrlFilters(this.getUrlFilters(attributes).filter(f => filter.field === f.field))`

Reply:
Fixed.